janetnim/bootcamp-challenge
# #157754177 Creating User/Admin Log in Page

**What does this PR do?**
Create a log in page

**Description of Task to be completed?**
A log in form should be designed with username/email and password fields

**How should this be manually tested?**
After adding the html and css files to the UI directory, open the login.html file in your browser to see if all components of the page appear and work as they should

**# What are the relevant pivotal tracker stories?**
#157754177
